### PR TITLE
Finished German Rewards

### PR DESCRIPTION
**Pull request by TryonHD**

### CODE REVIEW

File: translations/base-de.yaml [hunk 1]
```diff
@@ -524,52 +524,52 @@ storyRewards:
...
-        title: Chaining Extractor
-        desc: You have unlocked the <strong>chaining extractor</strong>! It can <strong>forward its resources</strong> to other extractors so you can more efficiently extract resources!
+        title: Extrahierer (Kette)
+        desc: Du hast den <strong>Extrahierer (Kette)</strong> freigeschaltet! Damit können die Ressourcen an den nächsten Extrahierer <strong>weitergeben</strong> um Ressourcen effizienter weiterzugeben.!
 
     reward_underground_belt_tier_2:
-        title: Tunnel Tier II
-        desc: You have unlocked a new variant of the <strong>tunnel</strong> - It has a <strong>bigger range</strong>, and you can also mix-n-match those tunnels now!
+        title: Tunnel Level II
+        desc: Du hast eine neue Variante des <strong>Tunnels</strong> freigeschaltet - Dieser hate eine  <strong>höhere Reichweite</strong> und du kannst beide Tunnel miteinander mischen!
```
Comment (tobspr) [inline translations/base-de.yaml:532]: *hate -> hat

File: translations/base-de.yaml [hunk 2]
```diff
@@ -524,52 +524,52 @@ storyRewards:
...
 
     reward_splitter_compact:
-        title: Compact Balancer
+        title: Kombinierer (kompakt)
         desc: >-
-            You have unlocked a compact variant of the <strong>balancer</strong> - It accepts two inputs and merges them into one!
+           Du hast eine kompakte Variante des <strong>Kombinierers</strong> freigeschaltet - Damit kannst du zwei förderbänder in eins vereinen!
```
Comment (tobspr) [inline translations/base-de.yaml:537]: *Förderbänder

File: translations/base-de.yaml [hunk 3]
```diff
@@ -524,52 +524,52 @@ storyRewards:
...
 
     reward_cutter_quad:
-        title: Quad Cutting
-        desc: You have unlocked a variant of the <strong>cutter</strong> - It allows you to cut shapes in <strong>four parts</strong> instead of just two!
+        title: Zerschneider (4-fach)
+        desc: Du hast eine neue Varienate des <strong>Zerschneiders</strong> freigeschaltet - Damit kannst du Formen in <strong>vier Teile</strong> anstatt zwei schneiden!
```
Comment (tobspr) [inline translations/base-de.yaml:541]: *Variante

File: translations/base-de.yaml [hunk 4]
```diff
@@ -524,52 +524,52 @@ storyRewards:
...
 
     reward_painter_double:
-        title: Double Painting
-        desc: You have unlocked a variant of the <strong>painter</strong> - It works as the regular painter but processes <strong>two shapes at once</strong> consuming just one color instead of two!
+        title: Färber (2-fach)
+        desc: Du hast eine neue Varienate des <strong>Färbers</strong> freigeschaltet - Hiermit kannst du <strong>zwei Formen auf einmal</strong> färben un verbrauchst nur eine Farbe!
```
Comment (tobspr) [inline translations/base-de.yaml:545]: See above, there are multiple :)

File: translations/base-de.yaml [hunk 5]
```diff
@@ -524,52 +524,52 @@ storyRewards:
...
-        title: Quad Painting
-        desc: You have unlocked a variant of the <strong>painter</strong> - It allows to paint each part of the shape individually!
+        title: Färber (4-fach)
+        desc: Du hast eine neue Varienate des <strong>Färbers</strong> freigeschaltet - Hiermit kannst du jedes Teil einer Form einzelnd färben!
 
     reward_storage:
-        title: Storage Buffer
-        desc: You have unlocked a variant of the <strong>trash</strong> - It allows to store items up to a given capacity!
+        title: Lager
+        desc: Du hast eine neue Varienate des <strong>Mülleimers</strong> freigeschaltet  - Hiermit kannst du überschuss bis zu einer gegebenen Kapazität lagern!
```
Comment (tobspr) [inline translations/base-de.yaml:553]: *Überschuss

File: translations/base-de.yaml [hunk 6]
```diff
@@ -524,52 +524,52 @@ storyRewards:
...
 
     reward_freeplay:
-        title: Freeplay
-        desc: You did it! You unlocked the <strong>free-play mode</strong>! This means that shapes are now randomly generated! (No worries, more content is planned for the standalone!)
+        title: Freispiel
+        desc: Du hast es geschaft! Du hast den <strong>Freispiel Modus</strong> freigeschaltet! Das heißt das Formen jetzt zufällig generiert werden! (Keine Sorge, mehr Inhalt ist für die Standaloneversion geplant!)
```
Comment (tobspr) [inline translations/base-de.yaml:557]: *Standalone-Version

File: translations/base-de.yaml [hunk 7]
```diff
@@ -524,52 +524,52 @@ storyRewards:
...
         desc: >-
-            This level gave you no reward, but the next one will! <br><br> PS: Better don't destroy your existing factory - You need <strong>all</strong> those shapes later again to <strong>unlock upgrades</strong>!
+            Dieses Level hat dir keine Belohnung gegeben, aber das nächste schon! <br><br> PS: Denk dran nicht deine alten Fabriken zu zerstören - Du wirst <strong>all</strong> später noch brauchen um <strong>Upgrades freizuschalten</strong>!
 
     no_reward_freeplay:
-        title: Next level
+        title: Nächstes Level
         desc: >-
-            Congratulations! By the way, more content is planned for the standalone!
+            Herzlichen Glückwunsch! Ach, und mehr Inhalt ist in der Standaloneversion geplant!
```
Comment (DerFeldspatz) [inline translations/base-de.yaml:572]: Übrigens/Apropos, in der Standalone-Version ist noch vieles mehr geplant!
(Inhalt klingt halt blöd)

**Comment by tobspr** [comment]
Is this still being worked on?

**Comment by DerFeldspatz** [comment]
Tbh, I don't know. Since I reviewed it, I'm going to copy most of the parts done here and open a new PR...

**Pull request closed without merging**